src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/Report.kt

```diff
@@ -283,7 +283,7 @@ class Report(
       report = this,
       sequence = nomisOffenderParty.sequence,
       prisonerNumber = nomisOffenderParty.offender.offenderNo,
-      firstName = nomisOffenderParty.offender.firstName,
+      firstName = nomisOffenderParty.offender.firstName!!,
```

Comment:
in practice, `NomisOffender::firstName` is always populated

Reply:
no longer nullable: https://github.com/ministryofjustice/hmpps-nomis-prisoner-api/pull/978